package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Order_Tracking",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^0.21.1",
+    "bootstrap": "^4.6.0",
+    "jsonwebtoken": "^8.5.1",
+    "react": "^17.0.2",
```

Comment:
Consider using fixed versions of your dependencies to avoid potential issues from automatic updates. This can be done by removing the caret (^) before the version number.